src/index.html

```diff
@@ -5,9 +5,80 @@
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Inter:wght@400;700;800&family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="./styles/main.scss">
   </head>
+
   <body>
-    <h1>Product cards</h1>
+    <div class="step">
+      <div class="step__items">
+        <a href="#">
+          <img
+            class="step__icon"
+            src="./images/imac.jpeg"
+            alt="Apple tv"
+          >
+        </a>
+
+        <h2 class="title">
+          APPLE A1419 iMac 27" Retina
+          <br>
+          5K Monoblock (MNED2UA/A)
```

Comment:
Do not hardcode text new line
```suggestion
          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
```